conditional/blueprints/co_op.py

```diff
@@ -43,6 +46,9 @@ def submit_co_op_form(user_dict=None):
                                  CurrentCoops.date_created > start_of_year()).first():
         return "User has already submitted this form!", 403
 
+    # Add to corresponding co-op ldap group
+    ldap_add_member_to_group(user_dict['account'], semester + '_coop')
```

Comment:
Do we do any validation of `semester`? If not, we probably should - don't want anyone trying to add themselves to `my_cool_custom_semester_coop`.